app/controllers/course/lesson_plan_items_controller.rb

```diff
@@ -1,11 +1,16 @@
 class Course::LessonPlanItemsController < Course::ComponentController
   before_action :load_lesson_plan_items, only: [:index]
+  # This can only be done with Bullet once Rails supports polymorphic +inverse_of+.
+  around_action :without_bullet, only: [:index]
```

Comment:
Bullet is only for development and test, will this break production ? 

And I'm not sure if we should put this in the code.